src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/SearchParameterRowGenerator.cs

```diff
@@ -15,18 +16,24 @@ internal abstract class SearchParameterRowGenerator<TSearchValue, TRow> : ITable
         where TRow : struct
     {
         private readonly bool _isConvertSearchValueOverridden;
+        private readonly bool _isInitializeOverridden;
+        private bool _isInitialized;
 
         protected SearchParameterRowGenerator(SqlServerFhirModel model)
         {
             EnsureArg.IsNotNull(model, nameof(model));
             Model = model;
             _isConvertSearchValueOverridden = GetType().GetMethod(nameof(ConvertSearchValue), BindingFlags.Instance | BindingFlags.NonPublic).DeclaringType != typeof(SearchParameterRowGenerator<TSearchValue, TRow>);
+            _isInitializeOverridden = GetType().GetMethod(nameof(Initialize), BindingFlags.Instance | BindingFlags.NonPublic).DeclaringType != typeof(SearchParameterRowGenerator<TSearchValue, TRow>);
```

Comment:
Can you add a comment about why we're doing this?

Reply:
I think this was a case of pointless optimizations. I'll remove.

|          Method |     Mean |     Error |    StdDev | Rank |
|---------------- |---------:|----------:|----------:|-----:|
|          Normal | 1.127 ns | 0.0582 ns | 0.0988 ns |    1 |
| MoreComplicated | 1.100 ns | 0.0247 ns | 0.0231 ns |    1 |